app/models/utils/load_testing/odk_submission_load_test.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module Utils
+  module LoadTesting
+    # Builds a JMeter test plan for submitting a lot of ODK form submissions.
+    #
+    # This test building process should be run on the server that is being targeted
+    # for the test because it will query the database while building the sample submission
+    # data (i.e. the XML for the ODK submission) for the test.
+    #
+    # The actual JMeter test will be run on a different server that does not have NEMO installed.
+    # It wil just know how to make a bunch of HTTP requests based on the options and data we give it.
+    class OdkSubmissionLoadTest < LoadTest
+      # Required `options`:
+      #   username: User doing the submitting
+      #   password: User's password
+      #   form_id: The ID of the form being submitted to.
+      #   mission_name: The name of the mission in which the form is being submitted
+      #
+      # The test currently knows how to handle the following question types:
+      #   text, long_text, barcode, integer, counter, decimal, location, select_one,
+      #   select_multiple, datetime, date, time
+
+      SUBMISSION_FILENAME = "submission.xml"
+
+      # Generates a bunch of test data to be used to make the test requests and store it in a CSV file.
+      def generate_test_data
+        form = Form.find(options[:form_id])
+        data = odk_submission(form)
```

Comment:
Are we just generating one submission? Can we do a few hundred perhaps? I'd rather not have them all be completely identical...

I also note we're not doing a CSV file. I guess that makes sense given the way we have to attach files to the submission. But I think to allow us to do multiple different submissions and keep it tractable we could use CSV and construct our own multipart request manually. See: https://stackoverflow.com/a/39020448/2066866

So to recap:

* CSV file full of lines of XML, each line being a full submission
* In the `test` block, we construct a multipart POST request setting `xml_submission_file` to the next line from the CSV.
* JMeter automatically wraps up to the top of the CSV file when it reaches the end.

Doable?